image/image.go

```diff
@@ -301,6 +301,13 @@ func createBundle(w walker, m *manifest, dest, rootfs string) error {
 			if err2 := os.MkdirAll(dest, 0755); err2 != nil {
 				return err2
 			}
+			defer func() {
+				if retErr != nil {
```

Comment:
Can we use `err` instead of `retErr`.  Otherwise I expect this will always be `nil` (unless we change to set `retErr` in the body of `createBundle`).  Or maybe Go does something magic with named returns to set them up before defers fire?

Reply:
when return ,go will assign the return value to `retErr`